tests/crud/test_hooks.py

```diff
@@ -0,0 +1,182 @@
+import json
+from unittest import TestCase
+
+from piccolo.columns import Integer, Varchar
+from piccolo.columns.readable import Readable
+from piccolo.table import Table
+from starlette.testclient import TestClient
+
+from piccolo_api.crud.endpoints import PiccoloCRUD
+from piccolo_api.crud.hooks import Hook, HookType
+
+
+class Movie(Table):
+    name = Varchar(length=100, required=True)
+    rating = Integer()
+
+    @classmethod
+    def get_readable(cls):
+        return Readable(template="%s", columns=[cls.name])
+
+
+async def set_movie_rating_10(row: Movie):
+    row.rating = 10  # type: ignore
+    return row
```

Comment:
You can use this `async def set_movie_rating_10(row: t.Any = Movie):` or `async def set_movie_rating_10(row):` to remove `# type: ignore`

Reply:
Another way around this is to do `row['rating'] = 10`.

Reply:
thanks! I just wanted to show that these objects can be used as regular piccolo objects, but we kindof do that in the documentation so it's probably not that important. I'll fix it by using dict-notation for now, feel free to comment on that!